lib/diagnostic.js

```diff
@@ -6,15 +6,23 @@
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
+
+const normalizeWords = function (text) {
+let words = text.split(/\s+/);
+  for (let i = 0; i < words.length; i++) {
+  let word = word [i];
```

Comment:
This should be ```let word = words[i];```.  It breaks because you are trying to assign ```word``` to itself before it has been declared.